src/components/CustomCoverageIcon/CustomCoverageIcon.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { Layout } from '@folio/stripes/components';
 
-export default function CustomCoverage() {
-  return (
-    <FormattedMessage id="ui-agreements.agreementLines.hasCustomCoverage">
-      {customCoverage => (
-        <Layout
-          className="flex"
-          data-test-custom-coverage
-          aria-label={customCoverage}
-        >
+
+const CustomCoverage = React.forwardRef((props, ref) => (
+  <FormattedMessage id="ui-agreements.agreementLines.hasCustomCoverage">
+    {() => (
+      <Layout
+        className="flex"
+        data-test-custom-coverage
+      >
+        <button type="button" ref={ref} aria-labelledby={props['aria-labelledby']}>
```

Comment:
This shouldn't be a `button`. Perhaps a `span` instead.

A button is a very specific thing that implies certain behaviour: it's clickable and does something when it's clicked. Custom coverage icons don't do either of those things so calling a button is bad form since it can be confusing to screen readers and can throw off things like CSS rules/selectors.

Reply:
Ah, this was on the suggestion of Rasmus, who mentioned that tooltips need to be contained inside focusable elements, is there something I should use instead of span in this case, or is there something I can do to span to make it focusable?

Reply:
Ahhh, makes sense. Personally I'd try setting `tabindex="-1"` on the span first. That makes the element focusable via programmatic means (which I assume is what's necessary for the Tooltip), but will not be tabbed through via the keyboard.

If that fails, then we can just go ahead and just use `button`, but add a comment above it to explain why this is in fact a button.

Reply:
That appears to have worked, closing this thread